cmd/tesseract/posix/README.md

```diff
@@ -1,8 +1,21 @@
 # POSIX static-ct server
 
-This directory contains an experimental `static-ct` server which uses Tessera's
-POSIX backend for storing the log.  It's not yet ready to be used as a
-production log!
+This directory contains a `static-ct` server which uses
```

Comment:
`tesseract` server? If this is a recent copy paste from somewhere else in the repo, let's get this fixed as well!

Reply:
This meant `static-ct` as in a server which complies with the `static-ct` spec.